pkg/deploy/api/validation/validation.go

```diff
@@ -50,6 +61,44 @@ func ValidateDeploymentConfigSpec(spec deployapi.DeploymentConfigSpec) field.Err
 	return allErrs
 }
 
+func handleEmptyImageReferences(template *kapi.PodTemplateSpec, triggers []deployapi.DeploymentTriggerPolicy) {
+	// if we have both an ICT defined and an empty Template->PodSpec->Container->Image field, we are going
+	// to modify this method's local copy (a pointer was NOT used for the parameter) by setting the field to a non-empty value to
+	// work around the k8s validation as our ICT will supply the image field value
+	containerEmptyImageInICT := make(map[string]bool)
+	for _, container := range template.Spec.Containers {
+		if len(container.Image) == 0 {
+			containerEmptyImageInICT[container.Name] = false
+		}
+	}
+	needToChangeImageField := false
+	if len(containerEmptyImageInICT) > 0 {
+		for _, trigger := range triggers {
+			// note, the validateTrigger call above will add an error if ImageChangeParams is nil, but
+			// we can still fall down this path so account for it being nil
+			if trigger.Type != deployapi.DeploymentTriggerOnImageChange || trigger.ImageChangeParams == nil {
+				continue
+			}
+
+			for _, container := range trigger.ImageChangeParams.ContainerNames {
+				needToChangeImageField = true
+				containerEmptyImageInICT[container] = true
```

Comment:
you are correct (non-empty changed to unset) ... in the end it doesn't matter but will address